.circleci/config.yml

```diff
@@ -175,10 +175,10 @@ jobs:
               --num-uniform-shards=50 \
               --app collect_app/build/outputs/apk/debug/*.apk \
               --test collect_app/build/outputs/apk/androidTest/debug/*.apk \
-              --device model=Pixel2,version=27,locale=en,orientation=portrait \
+              --device model=Pixel2,version=28,locale=en,orientation=portrait \
               --results-bucket opendatakit-collect-test-results \
               --environment-variables coverage=true,coverageFile=/sdcard/coverage.ec \
-              --directories-to-pull /sdcard --timeout 30m
+              --directories-to-pull /sdcard --timeout 20m
```

Comment:
We shouldn't need the 30m now we have sharding. Having it be 30m meant it was taking a long time to see timeout errors.